### PR TITLE
complete fft c2c on gpu in ND

### PR DESCRIPTION
**Pull request by jeff41404**
PR types
New features

PR changes
OPs

Describe
1. complete fft c2c on gpu in ND
2. complete normalization function

paddle.set_printoptions(precision=8, threshold=10000)
torch.set_printoptions(precision=8, threshold=10000)
np.set_printoptions(precision=8, threshold=10000)
np.random.seed(42)
x = (np.random.randn(4, 2, 2, 2, 2, 2) + 1j * np.random.randn(4, 2, 2, 2, 2, 2)).astype(np.complex64)
print(x)
print("numpy========================")
fft_x = np.fft.fftn(x)
print(fft_x)
print("pytorch======================")
xt = torch.tensor(x, device=torch.device('cuda:0'))
fft_xt = torch.fft.fftn(xt).cpu().numpy()
print(fft_xt)
print("paddle=======================")
xp = paddle.to_tensor(x)
fft_xp = paddle.tensor.fft.fftn(xp).numpy()
print(fft_xp)
print("allclose of paddle and pytorch is: %s" % np.allclose(fft_xt, fft_xp, rtol=1e-6, atol=0))
print("allclose of paddle and numpy is: %s" % np.allclose(fft_x, fft_xp, rtol=1e-6, atol=0))

### CODE REVIEW

**Review by iclementine — APPROVED** [review]
LGTM
